frontend/ReleaseDashboard/AssigneeFilter.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+    import { createEventDispatcher } from "svelte";
+    import Select from "svelte-select";
+    import User from "../Profile/User.svelte";
+    import { userList } from "../Stores/UserlistSubscriber";
+    import { filterUser } from "../Common/SelectUtils";
+
+    const dispatch = createEventDispatcher();
+    let users = {};
+    $: users = $userList;
```

Comment:
does it duplicate fetching users if parent component also fetch users?

Reply:
No, the store is global.